Add DisableSkipErrMetrics to disable ErrSkip metrics (#389)
XSAM/otelsql: CHANGELOG.md

## [Unreleased]

### Added

> - `DisableSkipErrMetrics` option skips the recording of `driver.ErrSkip` events if enabled. (#389)

Reviewer: ditto

> - `DisableSkipErrMeasurement` option suppresses the recording of `driver.ErrSkip` measurements as errors if enabled. (#389)

Reviewer: ```suggestion
- `DisableSkipErrMeasurement` option suppresses `driver.ErrSkip` as an error status in measurements if enabled. (#389)
```
